runtime/near-vm-logic/src/config.rs

```diff
@@ -292,6 +304,10 @@ impl Default for ExtCostsConfig {
         ExtCostsConfig {
             base: SAFETY_MULTIPLIER * 88420586,
             read_memory_base: SAFETY_MULTIPLIER * 861350075,
+            alt_bn128_g1_multiexp_base: SAFETY_MULTIPLIER * 50000000,
```

Comment:
These numbers are based on [EIP-1108](https://eips.ethereum.org/EIPS/eip-1108), 1 eth gas = 1e6 near gas.
So, for pairing constant cost is about 45_000_000_000, 
Byte cost is 34_000_000_000/(32*6) = 177_083_333.

Multiexponention constant cost is the same as for addition 150_000_000_000
Addition (element) cost is 6_000_000_000 / (254*2) = 11_811_023

Presented at the config values can differ a little.

Reply:
> 1 eth gas = 1e6 near gas.

I am not sure where this is coming from, I would assume it is coming from our EVM implementation but our EVM relationship of ETH to NEAR gas does not translate to the situation when we are adding a new host function. The way we estimate NEAR gas usage is by running `runtime-params-estimator` which calculates the exact number of CPU instructions that were used.

Reply:
I updated runtime-params-estimator, but there are no cost generators for
validator_stake_base and validator_total_stake_base.

So, measurment process paniced.

Reply:
@olonho (CC @bowenwang1996 ) Could we update `runtime-params-estimator` to use the values that are currently hardcoded for these fees, instead of computing them? I thnk @bowenwang1996 might have forgot to do it. It is currently blocking snjax.